frontend/src/routes/(authenticated)/project/[project_code]/AddProjectMember.svelte

```diff
@@ -37,6 +37,9 @@
       if (error?.byType('ProjectMembersMustBeVerified')) {
         return { usernameOrEmail: [$t('project_page.add_user.user_must_be_verified')] };
       }
+      if (error?.byType('ProjectMembersMustBeVerifiedForRole')) {
+        return { usernameOrEmail: [$t('project_page.add_user.manager_must_be_verified')] };
```

Comment:
I think this is more appropriate.
```suggestion
        return { role: [$t('project_page.add_user.manager_must_be_verified')] };
```

Reply:
Done in commit 3bff4b02b642680793b3e9f5fcde61dbfd1858a8.